### PR TITLE
hotfix: fincube #18 eliminating circular dependency in tables and removing foreign key constraints from DAO table

### PR DESCRIPTION
**Pull request by SampadSikder**
- Foreign key constraint hotfixes
- Remove uncessary columns from database

### CODE REVIEW

File: backend/dao-service/src/proposal-service/proposal-service.service.ts [hunk 1]
```diff
@@ -39,17 +39,13 @@ export class ProposalServiceService {
   }
 
 
-  async findOne(id: number, sub: string): Promise<any> {
+  async findAllProposals(sub: string): Promise<any> {
     const role = await this.getUserRole(sub);
     console.log(role);
```
Comment (FahimDev) [inline backend/dao-service/src/proposal-service/proposal-service.service.ts:44]: Try to use logger. Avoid console log in Dev merge

File: backend/dao-service/src/proposal-service/proposal-service.service.ts [hunk 2]
```diff
@@ -39,17 +39,13 @@ export class ProposalServiceService {
...
     if (role != 'MFS') {
       throw new UnauthorizedException("User does not have permission");
```
Comment (FahimDev) [inline backend/dao-service/src/proposal-service/proposal-service.service.ts:46]: Before Throwing error make sure to keep a log here so the distributed logging project can have a environment to test out.

**Review by FahimDev — CHANGES_REQUESTED** [review]
Fix the logger points.

File: backend/dao-service/src/proposal-service/proposal-service.service.ts (hunk 1 above)
Comment (SampadSikder) [reply backend/dao-service/src/proposal-service/proposal-service.service.ts]: This has been fixed

File: backend/dao-service/src/proposal-service/proposal-service.service.ts (hunk 2 above)
Comment (SampadSikder) [reply backend/dao-service/src/proposal-service/proposal-service.service.ts:48]: This has been fixed

**Review by FahimDev — APPROVED** [review]
LGTM

**Pull request merged**